### ISSUE TITLE
Exception when tabbing in a snippet inserted from Kite autocompletion

### ISSUE THREAD

**Issue by Moe-Kite** [spyder-ide/spyder#10376]
I tried to submit this issue from Spyder but the button `Submit to github` was still disabled even if I had the description completely filled

### What steps will reproduce the problem?

My buffer was
```
import requests
my_dict={}
my_dict["test_key2"] = 18

requests.pos$
```
$ being the position of my carret when I use the completion list.

Insert snippet for requests.post, selected the one with 2 arguments post(<url>, <my_dict>)

I then typed the url I wanted, then pressed tab to go to the second placeholder and got this crash windows.

Stacktrace:
```
  File "/data/anaconda/spyder/spyder/plugins/editor/widgets/codeeditor.py", line 1132, in process_signatures
    parameter_data = parameters[parameter_idx]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/moe/.conda/envs/spyder-dev2/lib/python3.7/logging/__init__.py", line 1025, in emit
    msg = self.format(record)
  File "/home/moe/.conda/envs/spyder-dev2/lib/python3.7/logging/__init__.py", line 869, in format
    return fmt.format(record)
  File "/home/moe/.conda/envs/spyder-dev2/lib/python3.7/logging/__init__.py", line 608, in format
    record.message = record.getMessage()
  File "/home/moe/.conda/envs/spyder-dev2/lib/python3.7/logging/__init__.py", line 369, in getMessage
    msg = msg % self.args
ValueError: incomplete format
Call stack:
  File "bootstrap.py", line 186, in <module>
    start.main()
  File "/data/anaconda/spyder/spyder/app/start.py", line 195, in main
    mainwindow.main()
  File "/data/anaconda/spyder/spyder/app/mainwindow.py", line 3628, in main
    mainwindow = run_spyder(app, options, args)
  File "/data/anaconda/spyder/spyder/app/mainwindow.py", line 3511, in run_spyder
    app.exec_()
  File "/data/anaconda/spyder/spyder/plugins/completion/plugin.py", line 138, in receive_response
    self.gather_and_send(request_responses)
  File "/data/anaconda/spyder/spyder/plugins/completion/plugin.py", line 189, in gather_and_send
    response_instance.handle_response(req_type, responses)
  File "/data/anaconda/spyder/spyder/plugins/editor/widgets/codeeditor.py", line 915, in handle_response
    handler(params)
  File "/data/anaconda/spyder/spyder/plugins/editor/widgets/codeeditor.py", line 1149, in process_signatures
    self.log_lsp_handle_errors("Error when processing signature")
  File "/data/anaconda/spyder/spyder/plugins/editor/widgets/codeeditor.py", line 946, in log_lsp_handle_errors
    logger.error('%', 1, stack_info=True)
Message: '%'
Arguments: (1,)
```
![image](https://user-images.githubusercontent.com/47991903/66521511-29e43780-eaa0-11e9-8798-1882bef27f74.png)

**Comment by Moe-Kite** [comment]
Gif of the issue : 
![spyder_crash](https://user-images.githubusercontent.com/47991903/66521810-ceff1000-eaa0-11e9-9495-76bad42da801.gif)

**Comment by ccordoba12** [comment]
@steff456, please verify if you can reproduce this one with the latest master.

**Comment by CAM-Gerlach** [comment]
![image](https://user-images.githubusercontent.com/17051931/66878521-d54d2a80-ef7f-11e9-9f4d-5ad1fdb38faa.png)

Regarding why you couldn't submit, you didn't type in a sufficiently descriptive title.

**Comment by gepcel** [comment]
It might be fixed by PR #10411 .

**Comment by Moe-Kite** [comment]
For the 15 characters, I actually thought that 15 character was the maximum (and not the minimum), it makes more sense as a minimum and explains why I couldn't validate even with the `Description complete; thanks!` message.

**Comment by ccordoba12** [comment]
> It might be fixed by PR #10411

Thanks for spotting this one @gepcel! You're totally right!

Closing as a duplicate of issue #10408
